packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -841,6 +843,34 @@ class WebViewClient extends JavaObject {
     String failingUrl,
   )? onReceivedError;
 
+  /// Notify host application that the given WebView's render process has exited.
+  ///
+  /// Multiple WebView instances may be associated with a single render process;
+  /// onRenderProcessGone will be called for each WebView that was affected.
+  final void Function(
+    WebView webView,
+    bool didCrash,
```

Comment:
This should pass back a `RenderProcessGoneDetail` with `didCrash` and `rendererPriorityAtExit` as fields. You can follow the example of `WebResourceRequest`.

Reply:
Changed the function to pass back a `RenderProcessGoneDetail`.